src/libcamera/pipeline/simple/simple.cpp

```diff
@@ -493,7 +493,7 @@ int SimpleCameraData::init()
 	MediaDevice *converter = pipe->converter();
 	if (converter) {
 		converter_ = ConverterFactoryBase::create(converter);
-		if (!converter_->isValid()) {
+		if (!converter_) {
```

Comment:
Yikes - definitely looks like a good spot here. Do we need to check that it's also valid to ensure construction was successful as well ?

I.e. it might need to be
if (!converter_ || !converter_->isValid())  { ?

Reply:
Hi @kbingham,
I'd asked this to Laurent and he said that it probably doesn't make much sense for the converter factory to return a non-null but invalid converter : )

Reply:
Gotcha, I'll use git-send-email to send to patch.

Reply:
Lets continue discussion on the list then. Thanks